e2e/features/submit-hearing-requirements.feature

```diff
@@ -32,7 +32,6 @@ Feature: Submit hearing requirements
     And I should see the text `The appeal is now going to a hearing. You need to submit the appellant's hearing requirements.`
     And I should see the case details
     And I should not see the hearing details
-    And I should see the legal representative details
```

Comment:
Is there any reason we do not want to see these details anymore?

Reply:
Added - should see the legal representative details